OpenEdXMobile/res/layout/item_course_date_block.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:binding="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+        <variable
+            name="dateBlock"
+            type="org.edx.mobile.model.course.CourseDateBlock" />
+
+        <variable
+            name="dateItemListener"
+            type="org.edx.mobile.interfaces.OnDateBlockListener" />
+
+        <variable
+            name="dateBlockList"
+            type="java.util.ArrayList&lt;org.edx.mobile.model.course.CourseDateBlock&gt;" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/item_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/white"
+        android:orientation="horizontal">
+
+
+        <View
+            android:id="@+id/line_above_dot"
+            android:layout_width="@dimen/width_hairline"
+            android:layout_height="0dp"
+            android:background="@drawable/black_line_vertical"
+            app:layout_constraintBottom_toTopOf="@+id/bullet"
+            app:layout_constraintEnd_toEndOf="@+id/bullet_today"
+            app:layout_constraintStart_toStartOf="@+id/bullet_today"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <ImageView
```

Comment:
we can use `View` instated of `ImageView`.  Also, update `DataBindingHelperUtils` methods accordingly.